dataflux_pytorch/dataflux_iterable_dataset.py

```diff
@@ -130,6 +132,11 @@ def __init__(
         self.bucket_name = bucket_name
         self.data_format_fn = data_format_fn
         self.config = config
+        if not self._has_permissions():
+            logging.warning(
```

Comment:
I wonder if we should return an error in this case and force the user to disable compose themselves, so that it's clear to them that composed download isn't happening. @bernardhan33 thoughts?

Reply:
Makes sense. So that if they still want the composed download enabled, they can manually add the permissions and try again.

Reply:
In case permission is missing, returning error.